src/components/Profile/Subscribe.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component, Fragment } from "react";
+import { connect } from "react-redux";
+import { Button, Header } from "semantic-ui-react";
+import { subscribeNotifications } from "../../redux/actioncreators/subscribeActions";
+
+export class UserSettings extends Component {
+  handleClick = e => {
+    this.props.subscribeNotifications();
+  };
+  render() {
+    const getNotifications = this.props.profile.get_notifications;
+    let subscriptionSegment = null;
+    if (getNotifications === true) {
+      subscriptionSegment = (
+        <Fragment>
+          <Header as="h2">Your are already subscribed to notifications</Header>
+          <p>
+            To unsubscribe, click the unsubscribe link on the notification
```

Comment:
You can add the link to the page (toggle for subscribe and unsubscribe) if possible instead of a user looking for it in the email.